src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -8,7 +8,58 @@ slug: /system/subsystems/behaviour
 
 The behaviour subsystem includes walk planning, soccer strategies, get up detection, head behaviour, script running and tuning, and kick behaviour.
 
-# Script Runner
+## Walk Behaviour
+
+The [Direct Walk Controller](https://github.com/NUbots/NUbots/tree/master/module/behaviour/skills/DirectWalkController) sends messages to the walk engine. It can send a message to enable or disable the walk engine, stop the walk engine, or send a walkcommand message to the walk engine.
```

Comment:
```suggestion
The [Direct Walk Controller](https://github.com/NUbots/NUbots/tree/master/module/behaviour/skills/DirectWalkController) sends messages to the walk engine. It can send a message to enable or disable the walk engine, stop the walk engine, or send a `WalkCommand`message to the walk engine.
```

Reply:
```suggestion
The [Direct Walk Controller](https://github.com/NUbots/NUbots/tree/master/module/behaviour/skills/DirectWalkController) sends messages to the walk engine. It can send a message to enable or disable the walk engine, stop the walk engine, or send a `WalkCommand` message to the walk engine.
```
missing a space